etc/bash_completion.d/yab

```diff
@@ -0,0 +1,16 @@
+# bash completion for yab executable
```

Comment:
please mention that this is based on the example from
https://godoc.org/github.com/jessevdk/go-flags#hdr-Completion

Reply:
Done